ports/stm32/boards/ARDUINO_OPTA/wifi_nvram_1dx.h

```diff
@@ -0,0 +1,49 @@
+static const uint8_t wifi_nvram_4343[] CYW43_RESOURCE_ATTRIBUTE =
```

Comment:
Is there a reason you need a custom nvram file?  Or can you use the one supplied by cyw43-driver?  If this custom one is needed then maybe add a comment explaining why.

Reply:
> Is there a reason you need a custom nvram file

Yes they all use a custom nvram file to disable the LPO.

Reply:
Ah, I see.  Very good.